docs/tasks.md

```diff
@@ -439,4 +439,256 @@ The following tables gives you an overview of the tasks in MTEB.
 | [YahooAnswersTopicsClassification](https://huggingface.co/datasets/yahoo_answers_topics) | ['eng'] | Classification | s2s | [Web] | {'test': 60000} | {'test': 346.35} |
 | [YelpReviewFullClassification](https://arxiv.org/abs/1509.01626) (Zhang et al., 2015) | ['eng'] | Classification | s2s | [Reviews] | {'test': 50000} |  |
 | [YueOpenriceReviewClassification](https://github.com/Christainx/Dataset_Cantonese_Openrice) (Xiang et al., 2019) | ['yue'] | Classification | s2s | [Reviews] | {'test': 6161} | {'test': 173.0} |
-<!-- TABLE END -->
\ No newline at end of file
+<!-- TASKS TABLE END -->
+
+<br/>
+
+## Task Count per Language
```

Comment:
I would probably move this above the first table or enclose both tables in a fold down menu (probably the best solution).

Reply:
Will do expand for both.

Reply:
Done.